lib/looker-sdk/client.rb

```diff
@@ -238,6 +238,32 @@ class Serializer < Sawyer::Serializer
       def encode(data)
         data.kind_of?(Faraday::UploadIO) ? data : super
       end
+
+      # slight modification to the base class' decode_has_value function to
+      # less permissive when decoding time values.
+      #
+      # See https://github.com/looker/helltool/issues/22037 for more details
+      def decode_hash_value(key, value)
```

Comment:
I'd like to see a github link to the exact method you are replacing here. That is very helpful as people upgrade stuff and decide if existing monkey patches are still appropriate or not.